custom_components/belgiantrain/__init__.py

```
@@ -1,52 +1,52 @@
+    from .data import BelgiantrainConfigEntry
 
+# TODO List the platforms that you want to support.
+# For your initial PR, limit it to 1 platform.
 PLATFORMS: list[Platform] = [
     Platform.SENSOR,
     Platform.BINARY_SENSOR,
     Platform.SWITCH,
 ]
 
 
+# TODO Update entry annotation
```

Ruff (0.8.2)

Missing author in TODO; try: `# TODO(<author_name>): ...` or `# TODO @<author_name>: ...`

(TD002)